contents/handbook/company/structure.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Small Teams
+sidebar: Handbook
+showTitle: true
+hideAnchor: true
+---
+
+PostHog is structured for speed, autonomy and innovation.
+
+Traditional organizations have big, separate functions. You have a product team, an engineering team, customer support, and so on. This slows things down, which stifles innovation - you have to get your boss to talk to someone else's boss to get work done. It also means that people can't really see the impact of their work.
+
+PostHog started off as a completely flat company with one big goal: to increase the number of successful products in the world.
+
+As we are getting bigger, we realized that it's harder for people to see the direct impact of their work, which reduces the sense of ownership.
+
+We have therefore introduced Small Teams. These are designed to each operate like a startup. 
+
+# How it works
+
+* A Small Team should never be more than ten people
+* A Small Team has an Accountable Person responsible for its performance - whoever is most appropriate depending on what the team is working on. Who would make the right founder for that bit of the company?
+* A Small Team must have (1) a customer, (2) a mission and (3) metrics
+* There may be certain functions where at our current stage we don't need a Small Team yet.
+* Each small team runs its own retrospective + sprint every week for 30 mins.
```

Comment:
Is the idea here that these sprints all replace the existing sprints we currently run?

Reply:
Yes, although I was on the fence if this would work on the grounds that it might feel silly this early, but I think it'd mean there'd be a more meaningful group discussion if it were a little smaller.

However, having slept on this, I think I've probably suggested too many teams for now, so slightly consolidating them until we're at > 2 pizza rule might make sense.

Reply:
Yeah I think fewer teams as @kpthatsme has outlined in groupings below would help, especially if folks want to sit in on multiple sprint meetings.

Reply:
I've just consolidated it a bit, but this is rough until we clarify if we're focussing on startups still or ie startups and scaleups that have no choice other than to use PostHog